pvlib/pvsystem.py

```diff
@@ -2029,13 +2029,6 @@ def retrieve_sam(name=None, path=None):
     KeyError
         If the provided ``name`` is not a valid database name.
 
-    Notes
-    -----
-    Files available at
-        https://github.com/NREL/SAM/tree/develop/deploy/libraries
```

Comment:
I think this link is worth retaining; it's the place to go to get the latest database file

Reply:
What's more, this docstring could benefit from a Examples section demonstrating how to use the raw URLs.